frameworks/person-escort-record/questions/medication-while-moving.yml

```diff
@@ -0,0 +1,31 @@
+type: radio
+question: Do they need to take any medication while moving?
+description: Medication while moving
+options:
+  -
+    label: "Yes"
+    value: "Yes"
+    followup:
+      - medication-carrier-while-moving
+    followup_comment:
+      label: Provide details of medication required while moving
+      hint: |
+        Ensure the following details are included:
```

Comment:
What about `Include the following details:`? I think it's better to use a verb we want someone to do at the start of the instruction rather than the end.

Reply:
The [Design System](https://design-system.service.gov.uk/components/error-message/#use-instructions-and-descriptions) follows a similar pattern but using the instruction at the start rather than using the word "Ensure". It feels superfluous in this instance.

Reply:
cc @Andy-Millar

Reply:
For the hint text - "Include the following details:" is fine

Reply:
Amended, although we have something similar here: https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks/blob/master/frameworks/person-escort-record/manifests/health-information.yml#L47

I left as is for now, but maybe for consistency we do the same as well